sdcm/provision/provisioner.py

```diff
@@ -80,6 +82,11 @@ class VmInstance:  # pylint: disable=too-many-instance-attributes
     image: str
     creation_time: datetime | None
     _provisioner: "Provisioner"
+    _node: Node = None
```

Comment:
`VmInstance` is common for all cloud providers, we shouldn't add libcloud dependencies here.
Can we do it differently?